packages/components/src/border-control/styles.ts

```diff
@@ -180,10 +180,10 @@ export const borderControlStylePicker = css`
 
 export const borderStyleButton = css`
```

Comment:
Have you tried using the `size="compact"` prop on the `Button` component, instead of adding style overrrides?